pkg/globals/application.go

```diff
@@ -1,12 +1,26 @@
 package globals
 
+import "os"
+
 const (
-	DDServiceName = "kubehound"
-	DDEnv         = "prod"
+	DDServiceName    = "kubehound"
+	DefaultDDEnv     = "dev"
+	DefaultComponent = "kubehound-ingestor"
 )
 
-func ProdEnv() bool {
-	return false
+func GetDDEnv() string {
+	env := os.Getenv("DD_ENV")
+	if env == "" {
+		return DefaultDDEnv
+	}
+
+	return env
+}
+
+func IsDDFormat() bool {
+	_, isDDEnv := os.LookupEnv("DD_ENV")
+
+	return isDDEnv
```

Comment:
Not a huge fan of IsDDFormat here, what format is it? (I know it's for the logs because this PR says it, but it's a global func)

Reply:
ok I just reuse what was in place. Did not like the `ProdEnv()` func. Will see if it is not too painful to add it to the config file instead.